Example/Custom Integration/SEPADebitExampleViewController.m

```diff
@@ -0,0 +1,85 @@
+//
+//  SEPADebitExampleViewController.m
+//  Custom Integration
+//
+//  Created by Cameron Sabol on 10/7/19.
+//  Copyright © 2019 Stripe. All rights reserved.
+//
+
+#import "SEPADebitExampleViewController.h"
+
+#import "MyAPIClient.h"
+
+@interface SEPADebitExampleViewController ()
```

Comment:
Intro blurb like some of the other examples + a link to the doc would be useful I think